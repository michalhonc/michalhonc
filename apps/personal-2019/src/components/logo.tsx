import * as React from "react";
import { Link } from "gatsby-plugin-intl";

import LogoImage from "../images/logo.svg";
import LogoBackground from "../images/logo-background.svg";

export const Logo: React.FC = () => {
  return (
    <div className="Header-logo">
      <div>
        <LogoBackground className="Header-logo-background" />
        <Link to="/" aria-label="Michal Honc Logo">
          <LogoImage className="Header-logo-foreground" />
        </Link>
      </div>
    </div>
  );
};
