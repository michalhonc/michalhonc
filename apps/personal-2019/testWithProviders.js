import React from "react";
import renderer from "react-test-renderer";
import { IntlContextProvider } from "gatsby-plugin-intl";

export const renderWithProviders = (component) =>
  renderer.create(
    <IntlContextProvider value={{ language: "en-US", routed: true }}>
      {component}
    </IntlContextProvider>
  );
