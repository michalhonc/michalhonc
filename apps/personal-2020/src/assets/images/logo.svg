<svg width="336" height="96" viewBox="0 0 336 96" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M31.2496 95.8316C30.9871 95.8316 30.7204 95.7816 30.4621 95.6775C29.3996 95.2441 28.8871 94.0275 29.3246 92.9608L66.8254 1.29218C67.2629 0.229653 68.4879 -0.278691 69.5421 0.154652C70.6046 0.587994 71.1171 1.80469 70.6796 2.87138L33.1788 94.54C32.8496 95.3441 32.0705 95.8316 31.2496 95.8316Z" fill="#FFDC4E"/>
<path d="M72.9172 79.1647C72.438 79.1647 71.9546 78.998 71.563 78.6605C70.6879 77.9105 70.5879 76.5979 71.338 75.7229L95.1718 47.914L71.3338 20.1009C70.5838 19.2301 70.6838 17.9134 71.5588 17.1634C72.4338 16.4134 73.7463 16.5134 74.4963 17.3884L99.4969 46.5557C100.168 47.3348 100.168 48.4849 99.4969 49.264L74.4963 78.4313C74.088 78.9188 73.5047 79.1647 72.9172 79.1647Z" fill="#242938"/>
<path d="M27.0829 79.1646C26.4953 79.1646 25.912 78.9188 25.5037 78.4355L0.503135 49.2682C-0.167712 48.489 -0.167712 47.339 0.503135 46.5598L25.5037 17.3925C26.2537 16.5175 27.5704 16.4175 28.4412 17.1675C29.3121 17.9134 29.4162 19.2301 28.6621 20.1009L4.82823 47.914L28.6662 75.7271C29.4162 76.5979 29.3162 77.9146 28.4412 78.6646C28.0454 78.998 27.562 79.1646 27.0829 79.1646Z" fill="#242938"/>
<path d="M130.409 64.1C131.474 64.1 132.387 64.3663 133.146 64.899C133.905 65.4317 134.478 66.1513 134.863 67.058C135.248 67.9533 135.441 68.9507 135.441 70.05C135.441 71.1607 135.248 72.1693 134.863 73.076C134.478 73.9713 133.905 74.6853 133.146 75.218C132.387 75.7393 131.474 76 130.409 76H126.227C126 76 125.808 75.9207 125.649 75.762C125.49 75.6033 125.411 75.4107 125.411 75.184V64.916C125.411 64.6893 125.49 64.4967 125.649 64.338C125.808 64.1793 126 64.1 126.227 64.1H130.409ZM130.239 74.436C131.395 74.436 132.262 74.028 132.84 73.212C133.418 72.3847 133.707 71.3307 133.707 70.05C133.707 68.7807 133.412 67.7323 132.823 66.905C132.245 66.0777 131.384 65.664 130.239 65.664H127.094V74.436H130.239ZM144.329 74.436C144.567 74.436 144.759 74.5153 144.907 74.674C145.065 74.8327 145.145 75.0197 145.145 75.235C145.145 75.4617 145.065 75.6487 144.907 75.796C144.759 75.932 144.567 76 144.329 76H138.413C138.186 76 137.993 75.9207 137.835 75.762C137.676 75.6033 137.597 75.4107 137.597 75.184V64.916C137.597 64.6893 137.676 64.4967 137.835 64.338C137.993 64.1793 138.186 64.1 138.413 64.1H144.329C144.567 64.1 144.759 64.1737 144.907 64.321C145.065 64.4683 145.145 64.661 145.145 64.899C145.145 65.1257 145.065 65.3127 144.907 65.46C144.759 65.596 144.567 65.664 144.329 65.664H139.28V69.132H143.564C143.802 69.132 143.994 69.2057 144.142 69.353C144.3 69.5003 144.38 69.6873 144.38 69.914C144.38 70.152 144.3 70.339 144.142 70.475C143.994 70.611 143.802 70.679 143.564 70.679H139.28V74.436H144.329ZM155.945 64.083C156.172 64.083 156.364 64.1567 156.523 64.304C156.693 64.4513 156.778 64.6327 156.778 64.848C156.778 64.9613 156.75 65.086 156.693 65.222L152.358 75.473C152.279 75.643 152.165 75.7733 152.018 75.864C151.871 75.9547 151.718 76 151.559 76C151.412 75.9887 151.27 75.9433 151.134 75.864C151.009 75.7733 150.907 75.6487 150.828 75.49L146.493 65.222C146.448 65.1313 146.425 65.0123 146.425 64.865C146.425 64.627 146.51 64.4343 146.68 64.287C146.85 64.1397 147.037 64.066 147.241 64.066C147.581 64.066 147.83 64.2473 147.989 64.61L151.695 73.365L155.197 64.593C155.276 64.4343 155.378 64.3097 155.503 64.219C155.639 64.1283 155.786 64.083 155.945 64.083ZM165.446 74.436C165.684 74.436 165.876 74.5153 166.024 74.674C166.182 74.8327 166.262 75.0197 166.262 75.235C166.262 75.4617 166.182 75.6487 166.024 75.796C165.876 75.932 165.684 76 165.446 76H159.53C159.303 76 159.11 75.9207 158.952 75.762C158.793 75.6033 158.714 75.4107 158.714 75.184V64.916C158.714 64.6893 158.793 64.4967 158.952 64.338C159.11 64.1793 159.303 64.1 159.53 64.1H165.446C165.684 64.1 165.876 64.1737 166.024 64.321C166.182 64.4683 166.262 64.661 166.262 64.899C166.262 65.1257 166.182 65.3127 166.024 65.46C165.876 65.596 165.684 65.664 165.446 65.664H160.397V69.132H164.681C164.919 69.132 165.111 69.2057 165.259 69.353C165.417 69.5003 165.497 69.6873 165.497 69.914C165.497 70.152 165.417 70.339 165.259 70.475C165.111 70.611 164.919 70.679 164.681 70.679H160.397V74.436H165.446ZM174.971 74.419C175.209 74.419 175.402 74.4927 175.549 74.64C175.708 74.7873 175.787 74.98 175.787 75.218C175.787 75.4447 175.708 75.6317 175.549 75.779C175.402 75.9263 175.209 76 174.971 76H169.225C168.987 76 168.789 75.9207 168.63 75.762C168.483 75.6033 168.409 75.4107 168.409 75.184V64.916C168.409 64.6893 168.488 64.4967 168.647 64.338C168.806 64.1793 169.01 64.1 169.259 64.1C169.486 64.1 169.678 64.1793 169.837 64.338C170.007 64.4967 170.092 64.6893 170.092 64.916V74.419H174.971ZM188.402 70.05C188.402 71.1947 188.152 72.2317 187.654 73.161C187.166 74.0903 186.486 74.827 185.614 75.371C184.741 75.9037 183.755 76.17 182.656 76.17C181.556 76.17 180.57 75.9037 179.698 75.371C178.825 74.827 178.139 74.0903 177.641 73.161C177.153 72.2317 176.91 71.1947 176.91 70.05C176.91 68.9053 177.153 67.8683 177.641 66.939C178.139 66.0097 178.825 65.2787 179.698 64.746C180.57 64.202 181.556 63.93 182.656 63.93C183.755 63.93 184.741 64.202 185.614 64.746C186.486 65.2787 187.166 66.0097 187.654 66.939C188.152 67.8683 188.402 68.9053 188.402 70.05ZM186.651 70.05C186.651 69.2 186.481 68.435 186.141 67.755C185.801 67.0637 185.325 66.5253 184.713 66.14C184.112 65.7433 183.426 65.545 182.656 65.545C181.885 65.545 181.194 65.7377 180.582 66.123C179.981 66.5083 179.511 67.0467 179.171 67.738C178.831 68.418 178.661 69.1887 178.661 70.05C178.661 70.9 178.831 71.6707 179.171 72.362C179.511 73.042 179.981 73.5803 180.582 73.977C181.194 74.3623 181.885 74.555 182.656 74.555C183.426 74.555 184.112 74.3623 184.713 73.977C185.325 73.5803 185.801 73.042 186.141 72.362C186.481 71.6707 186.651 70.9 186.651 70.05ZM195.299 64.1C195.9 64.1 196.455 64.2643 196.965 64.593C197.475 64.9103 197.877 65.3467 198.172 65.902C198.478 66.4573 198.631 67.0693 198.631 67.738C198.631 68.4067 198.478 69.0243 198.172 69.591C197.877 70.1577 197.475 70.6053 196.965 70.934C196.455 71.2627 195.9 71.427 195.299 71.427H192.103V75.184C192.103 75.4107 192.029 75.6033 191.882 75.762C191.735 75.9207 191.548 76 191.321 76C191.083 76 190.89 75.9263 190.743 75.779C190.607 75.6203 190.539 75.422 190.539 75.184V64.916C190.539 64.6893 190.618 64.4967 190.777 64.338C190.936 64.1793 191.128 64.1 191.355 64.1H195.299ZM195.299 69.88C195.605 69.88 195.888 69.7837 196.149 69.591C196.41 69.387 196.614 69.1207 196.761 68.792C196.92 68.4633 196.999 68.112 196.999 67.738C196.999 67.16 196.829 66.6727 196.489 66.276C196.16 65.868 195.764 65.664 195.299 65.664H192.103V69.88H195.299ZM207.464 74.436C207.702 74.436 207.895 74.5153 208.042 74.674C208.201 74.8327 208.28 75.0197 208.28 75.235C208.28 75.4617 208.201 75.6487 208.042 75.796C207.895 75.932 207.702 76 207.464 76H201.548C201.322 76 201.129 75.9207 200.97 75.762C200.812 75.6033 200.732 75.4107 200.732 75.184V64.916C200.732 64.6893 200.812 64.4967 200.97 64.338C201.129 64.1793 201.322 64.1 201.548 64.1H207.464C207.702 64.1 207.895 64.1737 208.042 64.321C208.201 64.4683 208.28 64.661 208.28 64.899C208.28 65.1257 208.201 65.3127 208.042 65.46C207.895 65.596 207.702 65.664 207.464 65.664H202.415V69.132H206.699C206.937 69.132 207.13 69.2057 207.277 69.353C207.436 69.5003 207.515 69.6873 207.515 69.914C207.515 70.152 207.436 70.339 207.277 70.475C207.13 70.611 206.937 70.679 206.699 70.679H202.415V74.436H207.464ZM219.489 74.64C219.602 74.7193 219.693 74.8213 219.761 74.946C219.829 75.0593 219.863 75.1783 219.863 75.303C219.863 75.541 219.783 75.7167 219.625 75.83C219.477 75.9433 219.296 76 219.081 76C218.865 76 218.684 75.9547 218.537 75.864C218.197 75.6713 217.919 75.3597 217.704 74.929C217.488 74.4983 217.381 73.8637 217.381 73.025C217.381 72.4583 217.211 72.039 216.871 71.767C216.531 71.495 216.089 71.359 215.545 71.359H212.111V75.184C212.111 75.422 212.043 75.6203 211.907 75.779C211.771 75.9263 211.595 76 211.38 76C211.119 76 210.892 75.9207 210.7 75.762C210.518 75.6033 210.428 75.4107 210.428 75.184V64.916C210.428 64.6893 210.507 64.4967 210.666 64.338C210.824 64.1793 211.017 64.1 211.244 64.1H216.021C216.633 64.1 217.194 64.253 217.704 64.559C218.214 64.8537 218.622 65.2673 218.928 65.8C219.234 66.3213 219.387 66.9107 219.387 67.568C219.387 68.2027 219.217 68.7977 218.877 69.353C218.537 69.897 218.1 70.305 217.568 70.577C217.987 70.7923 218.316 71.0927 218.554 71.478C218.803 71.8633 218.939 72.311 218.962 72.821C219.007 73.467 219.058 73.909 219.115 74.147C219.171 74.3737 219.296 74.538 219.489 74.64ZM215.987 69.948C216.27 69.9253 216.536 69.812 216.786 69.608C217.046 69.404 217.256 69.132 217.415 68.792C217.573 68.4407 217.653 68.0497 217.653 67.619C217.653 67.075 217.483 66.616 217.143 66.242C216.814 65.8567 216.412 65.664 215.936 65.664H212.111V69.948H215.987ZM236.353 74.589C236.591 74.8043 236.71 75.0367 236.71 75.286C236.71 75.4447 236.648 75.6033 236.523 75.762C236.398 75.9207 236.234 76 236.03 76C235.837 76 235.679 75.9547 235.554 75.864L233.99 74.572C232.925 75.6373 231.587 76.17 229.978 76.17C229.196 76.17 228.482 75.9943 227.836 75.643C227.201 75.2803 226.703 74.81 226.34 74.232C225.977 73.6427 225.796 73.025 225.796 72.379C225.796 71.5857 226.006 70.883 226.425 70.271C226.844 69.6477 227.473 69.1603 228.312 68.809C228.085 68.4803 227.915 68.1687 227.802 67.874C227.689 67.5793 227.632 67.2733 227.632 66.956C227.632 66.4007 227.774 65.8963 228.057 65.443C228.34 64.9783 228.748 64.61 229.281 64.338C229.814 64.066 230.431 63.93 231.134 63.93C231.916 63.93 232.602 64.0717 233.191 64.355C233.792 64.627 234.2 64.9783 234.415 65.409C234.494 65.5677 234.534 65.7433 234.534 65.936C234.534 66.1173 234.466 66.2703 234.33 66.395C234.205 66.5197 234.035 66.582 233.82 66.582C233.571 66.582 233.389 66.4913 233.276 66.31C233.038 65.97 232.743 65.7207 232.392 65.562C232.052 65.392 231.633 65.307 231.134 65.307C230.556 65.307 230.108 65.4543 229.791 65.749C229.485 66.0437 229.332 66.4233 229.332 66.888C229.332 67.296 229.479 67.7097 229.774 68.129C230.069 68.5483 230.488 69.0527 231.032 69.642L231.338 69.982C231.995 70.7073 232.8 71.495 233.752 72.345C234.047 71.767 234.251 71.1323 234.364 70.441C234.421 70.1123 234.647 69.948 235.044 69.948C235.271 69.948 235.452 70.0443 235.588 70.237C235.735 70.4183 235.792 70.6337 235.758 70.883C235.611 71.8123 235.327 72.6397 234.908 73.365C235.373 73.773 235.854 74.181 236.353 74.589ZM230.301 74.64C231.298 74.64 232.154 74.2943 232.868 73.603C231.655 72.5377 230.63 71.529 229.791 70.577C229.632 70.407 229.445 70.1917 229.23 69.931C228.584 70.1237 228.125 70.407 227.853 70.781C227.581 71.155 227.445 71.6083 227.445 72.141C227.445 72.583 227.547 72.991 227.751 73.365C227.955 73.739 228.272 74.045 228.703 74.283C229.134 74.521 229.666 74.64 230.301 74.64ZM248.263 64.1C249.329 64.1 250.241 64.3663 251 64.899C251.76 65.4317 252.332 66.1513 252.717 67.058C253.103 67.9533 253.295 68.9507 253.295 70.05C253.295 71.1607 253.103 72.1693 252.717 73.076C252.332 73.9713 251.76 74.6853 251 75.218C250.241 75.7393 249.329 76 248.263 76H244.081C243.855 76 243.662 75.9207 243.503 75.762C243.345 75.6033 243.265 75.4107 243.265 75.184V64.916C243.265 64.6893 243.345 64.4967 243.503 64.338C243.662 64.1793 243.855 64.1 244.081 64.1H248.263ZM248.093 74.436C249.249 74.436 250.116 74.028 250.694 73.212C251.272 72.3847 251.561 71.3307 251.561 70.05C251.561 68.7807 251.267 67.7323 250.677 66.905C250.099 66.0777 249.238 65.664 248.093 65.664H244.948V74.436H248.093ZM262.183 74.436C262.421 74.436 262.614 74.5153 262.761 74.674C262.92 74.8327 262.999 75.0197 262.999 75.235C262.999 75.4617 262.92 75.6487 262.761 75.796C262.614 75.932 262.421 76 262.183 76H256.267C256.04 76 255.848 75.9207 255.689 75.762C255.53 75.6033 255.451 75.4107 255.451 75.184V64.916C255.451 64.6893 255.53 64.4967 255.689 64.338C255.848 64.1793 256.04 64.1 256.267 64.1H262.183C262.421 64.1 262.614 64.1737 262.761 64.321C262.92 64.4683 262.999 64.661 262.999 64.899C262.999 65.1257 262.92 65.3127 262.761 65.46C262.614 65.596 262.421 65.664 262.183 65.664H257.134V69.132H261.418C261.656 69.132 261.849 69.2057 261.996 69.353C262.155 69.5003 262.234 69.6873 262.234 69.914C262.234 70.152 262.155 70.339 261.996 70.475C261.849 70.611 261.656 70.679 261.418 70.679H257.134V74.436H262.183ZM268.75 76.17C267.923 76.17 267.192 76.0567 266.557 75.83C265.923 75.592 265.316 75.2123 264.738 74.691C264.5 74.487 264.381 74.2547 264.381 73.994C264.381 73.79 264.461 73.6087 264.619 73.45C264.778 73.28 264.965 73.195 265.18 73.195C265.373 73.195 265.537 73.2573 265.673 73.382C266.127 73.8013 266.591 74.113 267.067 74.317C267.555 74.5097 268.104 74.606 268.716 74.606C269.43 74.606 270.031 74.4417 270.518 74.113C271.017 73.7843 271.266 73.3707 271.266 72.872C271.255 72.2827 271.006 71.8293 270.518 71.512C270.042 71.1833 269.311 70.9113 268.325 70.696C267.158 70.458 266.257 70.067 265.622 69.523C264.999 68.979 264.687 68.2253 264.687 67.262C264.687 66.5933 264.863 66.0097 265.214 65.511C265.566 65.001 266.053 64.61 266.676 64.338C267.3 64.066 268.002 63.93 268.784 63.93C269.487 63.93 270.15 64.0433 270.773 64.27C271.397 64.4967 271.901 64.797 272.286 65.171C272.547 65.3977 272.677 65.6413 272.677 65.902C272.677 66.106 272.598 66.2873 272.439 66.446C272.292 66.6047 272.111 66.684 271.895 66.684C271.737 66.684 271.606 66.6387 271.504 66.548C271.21 66.2533 270.802 66.004 270.28 65.8C269.759 65.596 269.26 65.494 268.784 65.494C268.025 65.494 267.419 65.6527 266.965 65.97C266.523 66.276 266.302 66.684 266.302 67.194C266.302 67.7493 266.523 68.1743 266.965 68.469C267.419 68.7637 268.087 69.013 268.971 69.217C269.855 69.4097 270.575 69.6363 271.13 69.897C271.697 70.1577 272.133 70.5203 272.439 70.985C272.745 71.4497 272.898 72.056 272.898 72.804C272.898 73.4613 272.711 74.045 272.337 74.555C271.975 75.065 271.476 75.4617 270.841 75.745C270.207 76.0283 269.51 76.17 268.75 76.17ZM276.724 75.167C276.724 75.405 276.639 75.6033 276.469 75.762C276.299 75.9207 276.101 76 275.874 76C275.625 76 275.421 75.9207 275.262 75.762C275.115 75.6033 275.041 75.405 275.041 75.167V64.916C275.041 64.6893 275.12 64.4967 275.279 64.338C275.438 64.1793 275.642 64.1 275.891 64.1C276.118 64.1 276.31 64.1793 276.469 64.338C276.639 64.4967 276.724 64.6893 276.724 64.916V75.167ZM288.278 69.744C288.505 69.744 288.692 69.8233 288.839 69.982C288.998 70.1407 289.077 70.3333 289.077 70.56V74.385C289.077 74.691 288.941 74.929 288.669 75.099C288.148 75.439 287.575 75.7053 286.952 75.898C286.329 76.0793 285.688 76.17 285.031 76.17C283.898 76.17 282.855 75.898 281.903 75.354C280.962 74.81 280.22 74.0733 279.676 73.144C279.132 72.2033 278.86 71.172 278.86 70.05C278.86 68.928 279.132 67.9023 279.676 66.973C280.22 66.0323 280.962 65.29 281.903 64.746C282.855 64.202 283.898 63.93 285.031 63.93C285.598 63.93 286.142 63.9923 286.663 64.117C287.196 64.2417 287.672 64.4287 288.091 64.678C288.227 64.7573 288.329 64.8537 288.397 64.967C288.465 65.0803 288.499 65.205 288.499 65.341C288.499 65.5677 288.42 65.766 288.261 65.936C288.114 66.0947 287.932 66.174 287.717 66.174C287.547 66.174 287.4 66.1343 287.275 66.055C286.538 65.715 285.79 65.545 285.031 65.545C284.215 65.545 283.467 65.7433 282.787 66.14C282.118 66.5367 281.586 67.0807 281.189 67.772C280.804 68.4633 280.611 69.2227 280.611 70.05C280.611 70.8773 280.804 71.6367 281.189 72.328C281.586 73.008 282.118 73.552 282.787 73.96C283.467 74.3567 284.215 74.555 285.031 74.555C285.416 74.555 285.819 74.5097 286.238 74.419C286.669 74.3283 287.037 74.2093 287.343 74.062V71.359H285.286C285.059 71.359 284.867 71.2853 284.708 71.138C284.549 70.9793 284.47 70.7867 284.47 70.56C284.47 70.322 284.549 70.1293 284.708 69.982C284.867 69.8233 285.059 69.744 285.286 69.744H288.278ZM300.251 64.1C300.467 64.1 300.642 64.1737 300.778 64.321C300.914 64.457 300.982 64.6327 300.982 64.848V75.167C300.982 75.405 300.903 75.6033 300.744 75.762C300.597 75.9207 300.404 76 300.166 76C300.042 76 299.917 75.983 299.792 75.949C299.679 75.9037 299.588 75.8413 299.52 75.762L292.839 67.058V75.252C292.839 75.4673 292.766 75.6487 292.618 75.796C292.482 75.932 292.307 76 292.091 76C291.876 76 291.7 75.932 291.564 75.796C291.428 75.6487 291.36 75.4673 291.36 75.252V64.916C291.36 64.678 291.434 64.4853 291.581 64.338C291.74 64.1793 291.933 64.1 292.159 64.1C292.295 64.1 292.426 64.1283 292.55 64.185C292.675 64.2303 292.771 64.2983 292.839 64.389L299.503 73.11V64.848C299.503 64.6327 299.571 64.457 299.707 64.321C299.855 64.1737 300.036 64.1 300.251 64.1ZM310.543 74.436C310.781 74.436 310.974 74.5153 311.121 74.674C311.28 74.8327 311.359 75.0197 311.359 75.235C311.359 75.4617 311.28 75.6487 311.121 75.796C310.974 75.932 310.781 76 310.543 76H304.627C304.401 76 304.208 75.9207 304.049 75.762C303.891 75.6033 303.811 75.4107 303.811 75.184V64.916C303.811 64.6893 303.891 64.4967 304.049 64.338C304.208 64.1793 304.401 64.1 304.627 64.1H310.543C310.781 64.1 310.974 64.1737 311.121 64.321C311.28 64.4683 311.359 64.661 311.359 64.899C311.359 65.1257 311.28 65.3127 311.121 65.46C310.974 65.596 310.781 65.664 310.543 65.664H305.494V69.132H309.778C310.016 69.132 310.209 69.2057 310.356 69.353C310.515 69.5003 310.594 69.6873 310.594 69.914C310.594 70.152 310.515 70.339 310.356 70.475C310.209 70.611 310.016 70.679 309.778 70.679H305.494V74.436H310.543ZM322.568 74.64C322.681 74.7193 322.772 74.8213 322.84 74.946C322.908 75.0593 322.942 75.1783 322.942 75.303C322.942 75.541 322.862 75.7167 322.704 75.83C322.556 75.9433 322.375 76 322.16 76C321.944 76 321.763 75.9547 321.616 75.864C321.276 75.6713 320.998 75.3597 320.783 74.929C320.567 74.4983 320.46 73.8637 320.46 73.025C320.46 72.4583 320.29 72.039 319.95 71.767C319.61 71.495 319.168 71.359 318.624 71.359H315.19V75.184C315.19 75.422 315.122 75.6203 314.986 75.779C314.85 75.9263 314.674 76 314.459 76C314.198 76 313.971 75.9207 313.779 75.762C313.597 75.6033 313.507 75.4107 313.507 75.184V64.916C313.507 64.6893 313.586 64.4967 313.745 64.338C313.903 64.1793 314.096 64.1 314.323 64.1H319.1C319.712 64.1 320.273 64.253 320.783 64.559C321.293 64.8537 321.701 65.2673 322.007 65.8C322.313 66.3213 322.466 66.9107 322.466 67.568C322.466 68.2027 322.296 68.7977 321.956 69.353C321.616 69.897 321.179 70.305 320.647 70.577C321.066 70.7923 321.395 71.0927 321.633 71.478C321.882 71.8633 322.018 72.311 322.041 72.821C322.086 73.467 322.137 73.909 322.194 74.147C322.25 74.3737 322.375 74.538 322.568 74.64ZM319.066 69.948C319.349 69.9253 319.615 69.812 319.865 69.608C320.125 69.404 320.335 69.132 320.494 68.792C320.652 68.4407 320.732 68.0497 320.732 67.619C320.732 67.075 320.562 66.616 320.222 66.242C319.893 65.8567 319.491 65.664 319.015 65.664H315.19V69.948H319.066Z" fill="#242938"/>
<path d="M124.149 15.2727V40H129.22V23.8452H129.425L135.825 39.8793H139.278L145.677 23.9055H145.882V40H150.953V15.2727H144.506L137.696 31.8864H137.406L130.597 15.2727H124.149ZM155.164 40H160.307V21.4545H155.164V40ZM157.748 19.0639C159.281 19.0639 160.537 17.8928 160.537 16.456C160.537 15.0312 159.281 13.8601 157.748 13.8601C156.226 13.8601 154.971 15.0312 154.971 16.456C154.971 17.8928 156.226 19.0639 157.748 19.0639ZM172.807 40.3622C177.745 40.3622 180.848 37.4645 181.089 33.2024H176.236C175.934 35.1825 174.63 36.2933 172.867 36.2933C170.464 36.2933 168.907 34.277 168.907 30.7273C168.907 27.2259 170.477 25.2216 172.867 25.2216C174.751 25.2216 175.958 26.4652 176.236 28.3125H181.089C180.872 24.0263 177.624 21.2131 172.783 21.2131C167.156 21.2131 163.679 25.1129 163.679 30.7997C163.679 36.4382 167.096 40.3622 172.807 40.3622ZM189.526 29.2784C189.538 26.8878 190.999 25.4872 193.1 25.4872C195.189 25.4872 196.42 26.8274 196.408 29.1094V40H201.552V28.1918C201.564 23.8452 199.004 21.2131 195.14 21.2131C192.327 21.2131 190.444 22.5412 189.599 24.7266H189.381V15.2727H184.383V40H189.526V29.2784ZM210.888 40.3501C213.629 40.3501 215.403 39.1548 216.309 37.4283H216.454V40H221.332V27.4915C221.332 23.0724 217.589 21.2131 213.46 21.2131C209.016 21.2131 206.094 23.3381 205.382 26.7188L210.139 27.1051C210.489 25.8736 211.588 24.968 213.435 24.968C215.186 24.968 216.188 25.8494 216.188 27.3707V27.4432C216.188 28.6385 214.92 28.7955 211.697 29.1094C208.026 29.4474 204.73 30.679 204.73 34.8203C204.73 38.4908 207.35 40.3501 210.888 40.3501ZM212.361 36.8004C210.779 36.8004 209.644 36.0639 209.644 34.6513C209.644 33.2024 210.839 32.4901 212.651 32.2365C213.773 32.0795 215.609 31.8139 216.224 31.4034V33.3714C216.224 35.3153 214.619 36.8004 212.361 36.8004ZM230.466 15.2727H225.322V40H230.466V15.2727ZM234.682 40H239.91V29.7855H250.523V40H255.739V15.2727H250.523V25.4751H239.91V15.2727H234.682V40ZM268.332 40.3622C273.959 40.3622 277.46 36.5107 277.46 30.7997C277.46 25.0526 273.959 21.2131 268.332 21.2131C262.706 21.2131 259.204 25.0526 259.204 30.7997C259.204 36.5107 262.706 40.3622 268.332 40.3622ZM268.356 36.3778C265.76 36.3778 264.432 33.9993 264.432 30.7635C264.432 27.5277 265.76 25.1371 268.356 25.1371C270.904 25.1371 272.232 27.5277 272.232 30.7635C272.232 33.9993 270.904 36.3778 268.356 36.3778ZM285.948 29.2784C285.96 26.8878 287.385 25.4872 289.461 25.4872C291.526 25.4872 292.77 26.8395 292.758 29.1094V40H297.901V28.1918C297.901 23.8693 295.366 21.2131 291.502 21.2131C288.749 21.2131 286.757 22.5653 285.924 24.7266H285.706V21.4545H280.805V40H285.948V29.2784ZM310.334 40.3622C315.272 40.3622 318.375 37.4645 318.617 33.2024H313.763C313.461 35.1825 312.157 36.2933 310.395 36.2933C307.992 36.2933 306.434 34.277 306.434 30.7273C306.434 27.2259 308.004 25.2216 310.395 25.2216C312.278 25.2216 313.485 26.4652 313.763 28.3125H318.617C318.4 24.0263 315.152 21.2131 310.31 21.2131C304.684 21.2131 301.206 25.1129 301.206 30.7997C301.206 36.4382 304.623 40.3622 310.334 40.3622Z" fill="#242938"/>
</svg>
