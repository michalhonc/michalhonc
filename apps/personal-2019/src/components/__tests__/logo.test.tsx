import * as React from "react";
import renderer from "react-test-renderer";
import { renderWithProviders } from "../../../testWithProviders";
import { Logo } from "../logo";

describe("logo", () => {
  it("renders correctly", () => {
    const tree = renderWithProviders(<Logo />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
