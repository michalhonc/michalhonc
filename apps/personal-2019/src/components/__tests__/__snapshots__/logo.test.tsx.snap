// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`logo renders correctly 1`] = `
<div
  className="Header-logo"
>
  <div>
    <test-file-stub
      className="Header-logo-background"
    />
    <a
      aria-label="Michal Honc Logo"
      href="/en-US/"
      onClick={[Function]}
    >
      <test-file-stub
        className="Header-logo-foreground"
      />
    </a>
  </div>
</div>
`;
